woocommerce/woocommerce-ios
# [Orders with Coupons M6/M7] Enable feature flag

#10838 must be merged first

## Description
This PR enables Orders with Coupons M6 and M7 in release builds.

There's a list of non-blocking (mainly UI) improvements that will be dealt with separately, these have been tracked here: https://github.com/woocommerce/woocommerce-ios/issues/10863

## Testing instructions
* Switch the WooCommerce run scheme from `Debug` to a `Release` build
* Run the app and observe that we're using the updated UI for product rows, where items are collapsable

| Before | After |
|--------|--------|
| ![Simulator Screenshot - iPhone 15 - 2023-10-12 at 12 01 53](https://github.com/woocommerce/woocommerce-ios/assets/3812076/386c8b84-44f8-4965-9ec3-63526bae3c83) | ![simulator_screenshot_233122DD-37DC-4B43-951A-956EF96053ED](https://github.com/woocommerce/woocommerce-ios/assets/3812076/2baf9813-4188-494b-bebb-f9e6435c663c) |
